infrastructure/helm/keycloak-setup.txt

```diff
@@ -0,0 +1,48 @@
+Config keycloak
+  Open browser at http://<LOCALIP>:<KEYCLOAK PORT>
+  Select Administration Console
+  Login with “user” / <PASSWORD>
+
+  create “quantumserverless” Realm
+    Dropdown “master”
+    “Create Realm”
+    Set “Realm name” “quantumserverless”
+    “Create”
+    "Realm settings" in the left Nav
+    "Action"
+      "Partial import"
+      "Browse"
+      Select "keycloakconfig.json"
+      Check all resources and Overwrite
+      "Import"
+      "Close"
+
+  Generate “rayclient” client secret
+    “Clients” in the left nav
+    Select "rayclient"
+    “Credentials” tab
+      "Regenerate" the "Client Secret"
+      "Yes"
+      Note “Client secret” value
+
+  Create “admin” user
+    “Users” in the left nav
+    “Create new user”
+    Set “username” “admin”
+    Set “Email” “admin@mysite.com”
+    Change “Email verified” to On
+    “Create”
+    “Credentials” tab
+      “Set Password”
+      Set “Password” and “Password confirmation” “password”
+      Change “Temporary” to Offf
+      “Save”
+      “Save password”
+
+Run ./login-setup-2.sh <CLIENT-SECRET> in the terminal
+
+Access to the ray dashboard
+  Start browser at “http://localhost/”
+    “Sign in with Keycloak OICD”
```

Comment:
I wasn't given a sign-in option, was taken straight to the dashboard...

Reply:
It must be something wrong.  I'll check

Reply:
I used the port, not the proxy, so it may have been something on my end...

Reply:
please try it in the `incognit` or `private window`.  The browser may remember something.